### PR TITLE
tp-qemu: only care peak value of throughput

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
ID:1212734

File: qemu/tests/ovs_qos.py [hunk 1]
```diff
@@ -173,15 +173,10 @@ def __get_vminfo():
...
-                                                     server_path,
-                                                     md5sum,
-                                                     netperf_link,
-                                                     port=info[-2],
-                                                     client=info[-3],
-                                                     password=info[-4],
-                                                     username=info[-5],
-                                                     compile_option=compile_option_server)
+                server = utils_netperf.NetperfServer(
+                    vm.get_address(), server_path, md5sum,netperf_link,
```
Comment (ypu) [inline qemu/tests/ovs_qos.py]: This line failed the Travis check. Please update it

**Comment by suqinhuang** [comment]
LGTM
Acked-by: Suqin Huang

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by ypu** [comment]
Thanks @xutian @suqinhuang and @CongSmile will merge it.

**Pull request merged**